src/makefile

```diff
@@ -15,21 +15,22 @@ else
 	CFLAGS+= -DHAVE_SODIUM_HKDF=1
 endif
 
-all: bin libopaque.$(SOEXT) tests$(EXT)
+all: test$(EXT)
```

Comment:
why does `test` need an extension? it's a phony target, not a binary.

Reply:
also it might be better to not have `tests` as a dep to `all` package maintainers will hate this when building.

Reply:
That's better. You're right. Neither `test` nor `tests` need an extension.

I also removed `test` from the `all` target in favor of `tests` which just builds.